projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -547,7 +547,7 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
             };
     }
 
-    public focused = this.active;
+    // public focused = this.active;
```

Comment:
@rkaraivanov Couldn't figure out if this is supposed to do anything. Found just one reference to it in template, but typescript was complaining this is initialized before `active` (which is completely true, it's initially undefined when this is assigned). It's not hidden, nor described.. should I just delete it or keep it hidden and deprecated for a bit?

Reply:
I guess deprecate it and remove it in a future version.